tests/model_forms/models.py

```diff
@@ -462,3 +462,16 @@ class Award(models.Model):
 
 class NullableUniqueCharFieldModel(models.Model):
     codename = models.CharField(max_length=50, blank=True, null=True, unique=True)
+
+
+# Map and MapSpot Model has been added to test the inlineformset forms.
+class Map(models.Model):
+    name = models.CharField(max_length=128)
+
+
+class MapSpot(models.Model):
+    map = models.ForeignKey('Map', models.CASCADE)
+    position = models.CharField(max_length=30)
+
+    class Meta:
+        unique_together = (('map', 'position'),)
```

Comment:
I'd be great if you could reuse the `Character` and `Award` models instead by setting `Award.Meta.unique_together = ('name', 'character')` because this will require the creation of two extra tables on every test suite runs which slows down its execution over time.

Reply:
I moved test to `tests/model_formsets/tests.py` and could reuse existing models: `Author` and `Book` which already had `unique_together` constraint